production/tanka/grafana-agent/config.libsonnet

```diff
@@ -63,12 +67,67 @@
 
           host_filter: $._config.agent_host_filter,
 
-          scrape_configs: $._config.kubernetes_scrape_configs,
+          scrape_configs:
+            if $._config.agent_host_filter then
+              $._config.kubernetes_scrape_configs + $._config.deployment_scrape_configs
+            else
+              $._config.kubernetes_scrape_configs,
           remote_write: $._config.agent_remote_write,
         }],
       },
     },
+    deployment_agent_config: self.agent_config {
```

Comment:
I'm not sure this is right, this would give the deployment the same set of scrape configs as the replicaset.

Reply:
#You're right, I missed adding the key. Thinking a bit more about it, I'm not going to try and be smart for the sake of saving 3 lines of duplication. I have added the config statically in the next commit.